dotcom-rendering/package.json

```diff
@@ -216,6 +222,7 @@
     "ts-jest": "^24.3.0",
     "ts-loader": "^9.3.0",
     "ts-unused-exports": "^8.0.0",
+    "tslib": "^2.4.1",
```

Comment:
Is requiring [`tslib`](https://github.com/Microsoft/tslib) a new thing?

Reply:
seems arbitary, removed